assets/section-main-product.css

```diff
@@ -648,6 +681,40 @@ a.product__text {
     right: 5rem;
     top: 2.2rem;
   }
+  .image_label {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 4px 8px;
```

Comment:
A good practice in prodution CSS is not to use px, but instead use rem, as most of the rest of this file already does.
See https://engageinteractive.co.uk/blog/em-vs-rem-vs-px for example